README.md

```diff
@@ -52,10 +54,17 @@ SCSS and Typescript.
 
 ## Credits
 
-Developed by <author-name> for <company-name>.
+Based on Piano Genie, created by Chris Donahue, Ian Simon, Sander Dieleman and part of the 
+[Google Magenta project](https://magenta.tensorflow.org/).
+
+Developed by Christian Stussak for IMAGINARY gGmbH.
 
 ## License
 
-Copyright © <year> <copyright-holder>
+Piano Genie:
```

Comment:
Same as above: The app is written from scratch and all third-party code comes for separately licensed libraries. If we list the PianoGenie license here, we would (technically) need to list all licenses and authors of all the libraries that are pulled in through NPM ...